src/Sulu/Bundle/MediaBundle/Resources/js/containers/MediaSelectionOverlay/MediaSelectionOverlay.js

```diff
@@ -54,7 +54,8 @@ class MediaSelectionOverlay extends React.Component<Props> {
     static createMediaListStore(
         collectionId: IObservableValue<?string | number>,
         excludedIds: IObservableValue<?Array<number>>,
-        locale: IObservableValue<string>
+        locale: IObservableValue<string>,
+        types: IObservableValue<?Array<string>>,
```

Comment:
Looks to me like the `types` variable is not an `IObservableValue`, but a just an `?Array<string>` instead